frontend/src/components/home.js

```diff
@@ -85,13 +86,17 @@ export default function Home(props) {
             <MenuIcon onClick={toggleNavWindow} />
           </Hidden>
           <Typography variant="h6" className={classes.title} align="center">
-            Intelligent Data Assistant
+			  Intelligent Data Science Chatbot
           </Typography>
+			<IconButton color="default" aria-label="info about the project" onClick={() => { window.open('https://softwarecampus.de/en/project/ida-intelligent-data-science-chatbot/', '_blank') }}>
```

Comment:
Use an anchor tag with the URL around the IconButton instead of window.open. Browsers consider window open as a pop-up and warn the users